src/modules/deposit/model/deposit.entity.ts

```diff
@@ -76,7 +76,8 @@ export type FeeBreakdown = {
 
 export type PartialDeposit = Pick<
   Deposit,
-  "id" | "status" | "depositDate" | "amount" | "destinationChainId" | "sourceChainId" | "feeBreakdown" | "depositTxHash" | "price" | "token"
+  "id" | "amount" | "sourceChainId" | "destinationChainId" | "status" | "depositTxHash" | "depositDate" |
+  "feeBreakdown" | "token" | "price" | "outputToken" | "outputTokenPrice"
```

Comment:
Not something actionable now, but I don't know if this is the place to put the `PartialDeposit` model as the name is too generic and its scope it's related tightly to the rewards service. 
Partial deposits are needed in different places and each use case has its own list of required fields, so I don't think we can have a single PartialDeposit fitting all the needs.